deployment/arm/deploy-region.sh

```diff
@@ -66,20 +73,32 @@ echo
 
 echo "Creating Regional Resource Group: $resourceGroupName"
 az group create \
---name $resourceGroupName \
---location $region
-
+    --name $resourceGroupName \
+    --location $region
+echo
```

Comment:
Is this empty `echo` needed?

Reply:
Yes - just to have a blank line in the output